src/binaryen.cpp

```diff
@@ -434,8 +467,10 @@ namespace hera {
     heraAssert(false, string("Unsupported import called: ") + import->module.str + "::" + import->base.str + " (" + to_string(arguments.size()) + "arguments)");
   }
 
+namespace {
+
 // NOTE: This should be caught during deployment time by the Sentinel.
-void BinaryenEngine::validate_contract(Module & module)
+void validate_contract(Module & module)
```

Comment:
@chfast Which is better, this in an anonymous namespace or an empty forward declaration of `wasm::Module` in `binaryen.h` - see the last commit for that.

Reply:
If possible, hide it from the header.

Reply:
It can stay in an anonymous namespace.